src/help/index.ts

```diff
@@ -127,8 +114,8 @@ export class Help extends HelpBase {
     const subTopics = this.sortedTopics.filter(t => t.name.startsWith(name + ':') && t.name.split(':').length === depth + 1)
     const subCommands = this.sortedCommands.filter(c => c.id.startsWith(name + ':') && c.id.split(':').length === depth + 1)
 
-    const title = command.description && this.render(command.description).split('\n')[0]
-    if (title) console.log(title + '\n')
+    const summary = this.summary(command) // command.description && this.render(command.description).split('\n')[0]
+    if (summary) console.log(summary + '\n')
```

Comment:
This is still pulling from the command description, not the `summary` but I can't understand why.

Reply:
This test exemplifies what I mean
https://github.com/oclif/core/pull/186